js/blocks/controls/fetch-input.js

```diff
@@ -63,28 +64,14 @@ class FetchInput extends Component {
 	}
 
 	updateSuggestions( value ) {
-		// Show the suggestions after typing at least 2 characters
-		// and also for URLs.
-		if ( value.length < 2 ) {
-			this.setState( {
-				showSuggestions: false,
-				selectedSuggestion: null,
-				loading: false,
-			} );
-
-			return;
-		}
-
 		this.setState( {
-			showSuggestions: true,
-			selectedSuggestion: null,
```

Comment:
This corrects a strange behaviour where suggestions loaded twice after selecting one and then refocusing the field.